README.md

```diff
@@ -31,11 +31,14 @@ Netable is built on a number of core principles we believe a networking library
 ### Standard Usage
 
 #### Make a new instance of `Netable`, and pass in your base URL:
+
 ```swift
 let netable = Netable(baseURL: URL(string: "https://api.thecatapi.com/v1/")!)
 ```
+See [here](#Additional Netable instance parameters) for information on adding additional instance parameters. 
```

Comment:
I think this [syntax is slightly off](https://gist.github.com/rachelhyman/b1f109155c9dafffe618), it should be #like-this-without-caps